warehouse/query-core/src/main/java/datawave/query/attributes/TypeAttribute.java

```diff
@@ -58,8 +58,8 @@ public Object getData() {
 
     @Override
     public void write(DataOutput out) throws IOException {
-        WritableUtils.writeString(out, datawaveType.getClass().toString());
-        writeMetadata(out);
+        WritableUtils.writeString(out, datawaveType.getClass().getName());
```

Comment:
This fixed a but that caused all deserialized attributes to resort to a NoOpType